tests/testthat/_snaps/as_tibble.md

```diff
@@ -0,0 +1,162 @@
+# as_tibble works with standard use
```

Comment:
This is a good test. Can we make the table shorter, so if it ever changes in the future it's easier to see the differences? Maybe one continuous variable, one dichotomous, and one categorical?